chroma-dependencies/Makefile

```diff
@@ -21,7 +21,7 @@ cleanrepo: TARGET=cleanrepo
 requirements::
 	# these are the ones provided by EL and EPEL and our own repo
 	set -e;                                                   \
-	{ echo "../chroma-externals/netaddr-0.7.5.zip";           \
+	{ echo "https://github.com/drkjam/netaddr/archive/rel-0.7.5.tar.gz"; \
```

Comment:
so what are the rules for the exceptions?  in a number of patches you've been quite insistent that these blocks line up, is there a reason why you can't align it here. there is no 80 character line restrictions in our source. if you are suggesting other areas of source code should adhere to a particular format then I think your changes should too.